### ISSUE TITLE
A11y_VisualStudioCodeClient_Problems Panel_Validate that Eslint populates the problems View_Keyboard:The keyboard focus does not shift to the View problem tool bar when it is open, and also it is not accessible via keyboard

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/vscode#159528]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG2.4.3;  

### Environment and OS details:
Visual Studio Code Version: 1.70.2 (user setup)
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:

1.  Open Visual studio code editor.
2. Press ctrl+shift+M
3. With a JavaScript file open, activate the ESLint extension and use keyboard navigation to arrive at the "error & warning" output button on bottom left pane.   
4. Navigate to the error output window ; hit ENTER when you arrive at each line item to go the destination of the error/warning
5.Tab till View Problem and press enter or use F8 
6. Verify that keyboard focus shifts to view problem or not

### Actual Result:
The keyboard focus does not shift to the View problem tool bar when it is open, and also it is not accessible via keyboard.

### Expected Result:
When the View problem tool bar is open, the keyboard focus should shift to that because it's having some interactive control in it.

### Attachment:
![A11y_VisualStudioCodeClient_Validate that Eslint populates the problems View_Keyboard](https://user-images.githubusercontent.com/100189407/187362097-d268782d-301e-4590-8072-2a7f51a0de00.gif)

**Comment by VijayaBharathyA** [comment]
Thanks  for information. its working
![159528](https://user-images.githubusercontent.com/100189407/188547329-b99742c6-d5a7-4b2b-85c1-5a630414de74.gif)
